source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/invitations/user-input.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {compose} from 'recompose';
+
+import { IAttributeProps } from '@lib/dom';
+import { i18nProps, withTranslations } from '@lib/i18n';
+interface IOwnProps {
+  onSubmit: (email: string) => void;
+}
+
+interface IState {
+  email: string;
+}
+
+type IProps = IOwnProps & IAttributeProps & i18nProps;
+class UserInput extends React.Component<IProps, IState>{
+  state = {email: ''};
+
+  didType = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const email = e.target.value;
+    this.setState({ email });
+  }
+  onSubmit = (e: React.MouseEvent<HTMLInputElement>) => {
+    const {onSubmit, t} = this.props;
+    const {email} = this.state;
+    onSubmit(email);
+  }
+
+  public render() {
+    const {t} = this.props;
+    const {email} = this.state;
+    return (<div className="flex justify-content-space-between">
```

Comment:
this should be a from with an onSubmit hook so that hitting enter also does the submit

Reply:
👍